PT/asciidoc/src/03_context_and_scope.adoc

```diff
@@ -0,0 +1,78 @@
+ifndef::imagesdir[:imagesdir: ../images]
+
+[[section-context-and-scope]]
+== Contexto e Escopo
+
+
+ifdef::arc42help[]
+[role="arc42help"]
+****
+.Conteúdo
+Contexto e escopo - como o nome sugere - delimita seu sistema (ou seja, seu escopo) de todos os seus parceiros de comunicação
+(sistemas e usuários vizinhos, ou seja, o contexto do seu sistema). Ele especifica, portanto, as interfaces externas.
+
+Se necessário, diferencie o contexto de negócios (entradas e saídas específicas do domínio) do contexto técnico (canais, protocolos, hardware).
+
+.Motivação
+As interfaces de domínio e as interfaces técnicas para parceiros de comunicação estão entre os aspectos mais críticos do seu sistema. Certifique-se de entendê-las completamente.
+
+.Forma
+Várias opções:
+
+* Diagramas de contexto
+* Listas de parceiros de comunicação e suas interfaces.
```

Comment:
Same as above. From "Listas de parceiros de comunicação e suas interfaces." to "Listas de componentes externos e suas interfaces."

Reply:
Changed to "componentes externos"